src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -33,19 +44,30 @@ const ChatItemMyTextV2 = ({
   messageType,
   data
 }: ChatItemMyTextProps) => {
-  const {setReplyPreview, onContextMenuPressed} = useMessageHook();
+  const {
+    setReplyPreview,
+    onContextMenuPressed,
+    pulseAnimation,
+    animatedBubbleStyle,
+    animatedPulseStyle
+  } = useMessageHook();
+  const scrollContext = React.useContext(ScrollContext);
+  const swipeableRef = React.useRef<Swipeable>(null);
 
-  const swipeableRef = React.useRef<Swipeable | null>(null);
-  const bubblePosition = useSharedValue(0);
+  React.useEffect(() => {
+    if (scrollContext?.selectedMessageId === data?.id) {
+      pulseAnimation.value = withSequence(
+        withDelay(300, withTiming(1.1, {duration: 200})),
+        withTiming(1, {duration: 200})
+      );
+      scrollContext?.setSelectedMessageId(null);
+    }
+  }, [scrollContext?.selectedMessageId]);
 
-  const isReply = messageType === 'reply';
-  const isReplyPrompt = messageType === 'reply_prompt';
+  const isReply = messageType === MESSAGE_TYPE_REPLY;
+  const isReplyPrompt = messageType === MESSAGE_TYPE_REPLY_PROMPT;
   const isShowUserInfo = !isContinuous || isReplyPrompt || isReply;
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `pulseAnimation` logic has been updated to use the `withSequence`, `withDelay`, and `withTiming` functions from 'react-native-reanimated'. This change seems to add a delay and sequence to the pulse animation. However, it's important to ensure that this doesn't introduce any performance issues or unwanted visual effects. If possible, test this change thoroughly on different devices and under various conditions to ensure it behaves as expected.



<!-- This is an auto-generated comment by OSS CodeRabbit -->